README.md

```diff
@@ -1,4 +1,4 @@
 ## Weather-API (non-commercial)
 
-https://developer.srgssr.ch/apis/srgssr-weather
+[https://developer.srgssr.ch/apis/srgssr-weather](https://developer.srgssr.ch/api-catalog/srf-weather/)
```

Comment:
Let's just update the URL instead of making a link out of the old URL.

Reply:
Mistake on my end to leave the old link in it - should be updated